RELEASING.md

```diff
@@ -51,20 +52,55 @@
 # Using the release script
 
 The python script ``release.py`` in the scripts directory can be used to automate
-the release process for patch releases and release candidates after the first one has
-been made.
+the release process as long as the branch to release already exists.
+
+To use it, please first install the script dependencies listed in test-requirements.
+
+## Patch releases or rc>1
+To release a patch release or a release candidate after rc1, first ensure you're
+checked out in the proper branch, and then run the release.py script like so:
+```
+./release.py --github-token YOUR_TOKEN --tag <TAG> --commit <COMMIT>
+```
+
+If your environment variables contain RPC_GITHUB_TOKEN, you can even run it like
+```
+./release.py --tag <TAG> --commit <COMMIT>
+```
+
+## Other versions
+The guess work currently done for patch and rc>1 isn't applicable for other versions.
```

Comment:
As mentioned, the use of 'guess', gives the wrong impression of what is happening.